DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -101,16 +142,255 @@ void FillBindingData(
     }
 }
 
+DmlDispatchable::BindPoints DmlDispatchable::GetBindPoints() const
+{
+    if (m_isSerializedGraph)
+    {
+        return m_bindPoints; 
+    }
+    else
+    {
+        const auto& desc = std::get<Model::DmlDispatchableDesc>(m_desc);
```

Comment:
You don't need to recalculate the bindingPoints for DmlDispatchable. It is already present in DmlDispatchableDesc. Thus you don't need this seperate method GetBindPoints.

Reply:
done